src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Docs.Build
 {
     internal static class OpsConfigLoader
     {
-        public static bool TryLoad(string docsetPath, string branch, out JObject result)
+        public static JObject TryLoad(string docsetPath, string branch)
```

Comment:
should this method be renamed to `Load` since the return type is not `bool` anymore

Reply:
yep, that sounds reasonable.